Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-auth.tsx

```diff
@@ -39,6 +39,7 @@ const defaultState = {
         permissions: { canAccessAdministration: false } as IUserPermissionsType,
         isInRole: false,
         isAdmin: false,
+        isLecturer: false,
```

Comment:
Why is this used for?

Reply:
It was added in the initial implementation and development of the profile page. Only usage I see of it is this method for setting the internal user: 

<img width="744" alt="image" src="https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/100528439/950ec5a1-76d0-4cf8-87eb-f060a256406d">

Reply:
It was added in the initial implementation and development of the profile page. I rewrote and deleted a big part of it's logic but this one has been left out. Deleting it doesn't seem to change anything so I am removing this property from the default state and the interface/type